packages/workflow/src/Cron.ts

```diff
@@ -0,0 +1,68 @@
+interface BaseTriggerTime<T extends string> {
```

Comment:
Let's tighten the constraint to a union of the admissible values?

Reply:
that's automatically done by the union type that's exported.

Reply:
What I meant is that internally, the constraint is too lax about what the base and extending interfaces accept:
```ts
BaseTriggerTime<'anything'>
EveryXTrigger<'anything'>
```
Minor thing!